Dockerfile

```diff
@@ -1,12 +1,12 @@
-# This Docker file is intended for the CI
-# A prerequisite is a published application in the .build/release  
-
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build-env
```

Comment:
Uses an intermediary container to build OC in the same OS type (Windows/Linux) than Docker runs.